package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "jupyterlab-judge",
-    "version": "1.26.0",
+    "version": "1.26.1-rc1",
```

Comment:
I released version 1.26.1-rc1 for testing purposes.
Once this PR is merged, I will release version 1.26.1.